tests/php/test-amp-gallery-embed-handler.php

```diff
@@ -26,6 +26,13 @@ public static function tearDownAfterClass() {
 		AMP_Options_Manager::update_options( self::$original_amp_options );
 	}
 
+	public function setUp() {
+		parent::setUp();
+
+		register_theme_directory( ABSPATH . 'wp-content/themes' );
+		delete_site_transient( 'theme_roots' );
+	}
```

Comment:
Should the directory be unregistered as done in `Test_AMP_Helper_Functions::tearDown`? In reality, this same code is now copied across multiple tests and it warrants factoring out into a trait.

Reply:
Addressed in https://github.com/ampproject/amp-wp/pull/5159/commits/f3707a1bad0d05cea8ef93ac41ca7854c70be8db.